src/ucm/mmap/install.c

```diff
@@ -160,13 +160,24 @@ ucm_fire_mmap_events_internal(int events, ucm_mmap_test_events_data_t *data)
                        data, (void)sbrk(-sbrk_size));
     }
 
-    if (events & UCM_EVENT_MADVISE) {
+    if (events & (UCM_EVENT_MADVISE|UCM_EVENT_VM_UNMAPPED)) {
         UCM_FIRE_EVENT(events, UCM_EVENT_MMAP|UCM_EVENT_VM_MAPPED, data,
                        p = mmap(NULL, ucm_get_page_size(), PROT_READ|PROT_WRITE,
                                 MAP_PRIVATE|MAP_ANON, -1, 0));
         if (p != MAP_FAILED) {
             UCM_FIRE_EVENT(events, UCM_EVENT_MADVISE, data,
-                           madvise(p, ucm_get_page_size(), MADV_NORMAL));
+                           madvise(p, ucm_get_page_size(),
+#if HAVE_DECL_MADV_REMOVE
```

Comment:
can we reuse some code with ucm_madvise()?
maybe if none of the MADV_xx events which actually release memory is not defined - don't test it, since we would ne be able to handle it anyway?

Reply:
nice catch BTW! we should probably add it to gtest

Reply:
If we skip this test based on MADV_ macro then we have to add such preprocessings into all locations where event groups are defined and used

Reply:
btw why not just use MADV_DONTNEED?

Reply:
MADV_DONTNEED doesn't free pages - didn't expect that it produced event... fixed.
also added gtset for madvise (and bit updated gtest infra for malloc hooks)